src/Mapping/ClassMetadata.php

```diff
@@ -1124,14 +1125,12 @@ private function validateAndCompleteTypedFieldMapping(array $mapping): array
     /**
      * Validates & completes the basic mapping information based on typed property.
      *
-     * @param array{type: self::ONE_TO_ONE|self::MANY_TO_ONE|self::ONE_TO_MANY|self::MANY_TO_MANY, fieldName: string, targetEntity?: class-string} $mapping The mapping.
+     * @param array{type: self::ONE_TO_ONE|self::MANY_TO_ONE|self::ONE_TO_MANY|self::MANY_TO_MANY, fieldName: string, targetEntity?: class-string, joinColumns: array<int, array<string, mixed>>|null} $mapping The mapping.
```

Comment:
Please use linebreaks here as well

```suggestion
     * @param array{
     *     type: self::ONE_TO_ONE|self::MANY_TO_ONE|self::ONE_TO_MANY|self::MANY_TO_MANY, 
     *    fieldName: string,
     *    targetEntity?: class-string,
     *    joinColumns: array<int, array<string, mixed>>|null
     * } $mapping The mapping.
```